javascript/ql/src/semmlecode.javascript.dbscheme

```diff
@@ -915,20 +918,22 @@ case @json_value.kind of
 
 @json_parent = @json_object | @json_array | @file;
 
+@json_locatable = @json_value | @json_parse_error;
+
 // locations
 @ast_node = @toplevel | @stmt | @expr | @property | @typeexpr;
 
 @locatable = @file 
     | @ast_node
     | @comment
     | @line
-    | @js_parse_error | @regexp_parse_error | @json_parse_error
+    | @js_parse_error | @regexp_parse_error
     | @regexpterm
-    | @json_value
+    | @json_locatable
     | @token
     | @cfg_node
     | @jsdoc | @jsdoc_type_expr | @jsdoc_tag
-    | @yaml_node | @yaml_error
+    | @yaml_locatable
     | @xmllocatable;
```

Comment:
Might as well include `@configLocatable` now that you included it in the dbscheme.

Reply:
Good idea.

Reply:
Done.

Reply:
Done.